lib/action_controller/serialization.rb

```diff
@@ -30,7 +30,7 @@ def get_serializer(resource, options = {})
         serializable_resource.serialization_scope ||= serialization_scope
         serializable_resource.serialization_scope_name = _serialization_scope
         begin
-          serializable_resource.adapter
+          serializable_resource.tap(&:adapter)
```

Comment:
What I mean is, if I change the `serializable_resource.tap(&:adapter)` to `serializable_resource` any specs will be broken? I wrote a [spec to test the instrumentation on the controller](https://github.com/bf4/active_model_serializers/commit/eaa9c6b084eb665f194570ed3a95094979ae7270#diff-f290b93cc1e01ae4a2c31174202b50eeR424) I think we need a spec to check the behavior we are doing here.